README.md

```diff
@@ -1 +1,46 @@
-# Conference-Page
\ No newline at end of file
+# Conference-Page
+
+> First Capstone Project From Microverse
+
+## Built With
+
+- HTML5
+- CSS3
+- JavasScript
+- Bootstrap 5
+
+## Live Demo (if available)
+
+Not available
+
+### Deployment
+
+https://swarzstein.github.io/Conference-Page/
+
+
+## Authors
+
+👤 Juan Zorrilla
+
+- GitHub: [@Swarzstein](https://github.com/Swarzstein)
+- Twitter: [@JFox_XCI](https://twitter.com/JFox_XCI)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/juan-a-zorrilla/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
Please, include in your README an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
- [ ] Your README is well documented 💯 but I would suggest giving appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) :wink: :pushpin:

![-1](https://user-images.githubusercontent.com/45231276/181178654-ec00dfb0-1990-40d1-b370-0f6dc8f01606.PNG)